bundle/android/strings.xml

```diff
@@ -179,7 +180,6 @@
     <string name="gutenberg_native_failed_to_save_files_please_tap_for_options" tools:ignore="UnusedResources">Failed to save files.\nPlease tap for options.</string>
     <string name="gutenberg_native_failed_to_upload_files_please_tap_for_options" tools:ignore="UnusedResources">Failed to upload files.\nPlease tap for options.</string>
     <string name="gutenberg_native_featured" tools:ignore="UnusedResources">Featured</string>
-    <string name="gutenberg_native_featured_image" tools:ignore="UnusedResources">Featured Image</string>
```

Comment:
The removal of this string is expected as it's now also used in the web version ([reference](https://github.com/WordPress/gutenberg/blame/291c2c27a9cf7b024e1044331c7c7cfa94b7861f/packages/edit-site/src/components/page-pages/index.js#L81)). Hence, it will be translated as part of the Gutenberg plugin.